README.md

```diff
@@ -15,13 +15,15 @@ The Node.js Collection is a [Medium](https://medium.com/) publication which serv
 We have list of people who review the post before publishing. If the post is specific to some technical topic, we seek review from someone who has knowledge of that domain.
 
 ### How to become an editor?
-We are looking for more reviewers. If you are interested, please open an issue in [nodejs-collection repo](https://github.com/nodejs/nodejs-collection/issues) with your details and the team will review it.
+We are looking for more reviewers. If you are interested, please open an issue in [nodejs-collection repo](https://github.com/nodejs/nodejs-collection/issues) with these details and the team will review it.
+- Name
+- Experties (i.e IoT, Node.js Modules, cli, APIs etc)
+- How much time you can give per week. (i.e 1-2 hours)
 
 ### Benefits of being an editor
-- Your name will be added in the [current editors list](https://github.com/nodejs/nodejs-collection#current-editors).
+- Your name will be added to the [current editors list](https://github.com/nodejs/nodejs-collection#current-editors).
 - You'll be added to [editors list on Medium](https://medium.com/the-node-js-collection/about).
-- You'll receive discount to attend OpenJS Foundation Conference.
-
+- Possibility to become a member of Node.js Project and avail benefits like [Travel Fund](https://github.com/nodejs/admin#travel-fund).
```

Comment:
This is a benefit of being a member of the project.

If being an editor includes being a member of the project, this is inherently true and phrasing it like this is slightly misleading since it's something that comes inherently anyway rather than a specific benefit of this work.

If being an editor does not include being a member of the project, then this is also slightly misleading since there is additional work above and beyond being an editor that this requires.

Reply:
Being an editor will not include being project member. I think we should keep it nomination based. Like [here](https://github.com/nodejs/node/blob/master/GOVERNANCE.md#collaborator-nominations).
Once we see, someone is actively reviewing posts for 2,3 months, we can nominate them to become part of Node.js Collection ( -> Community Committee -> Node.js Project).

I also don't want to overdo it in the start. Let's go with an easy and simple process, once we have enough editors, we can even put this on hold.

Point to mention Travel Fund here was that people know what benefits would they get in return. People who are not part of the project already, may not even know that there is a travel fund benefit.

Reply:
In the next PR, I'll introduce a custom logger as discussed in slack. For this PR, ignore it.

Reply:
>Point to mention Travel Fund here was that people know what benefits would they get in return.

This is a good thing to include for the relevant groups. I do think it's important though to include it in the correct place, and, as you said, this is not something that people _will_ get but something they _could_ get if they stick around. 

>Once we see, someone is actively reviewing posts for 2,3 months, we can nominate them to become part of Node.js Collection ( -> Community Committee -> Node.js Project).

Is this documented anywhere? If so, I think that is the most appropriate place to include that context since it's the context in which it is accurate.